seekers/__init__.py

```diff
@@ -1,14 +1,5 @@
 from __future__ import annotations
 
 from .colors import Color
-from .seekers_types import (
-    Config,
-    Vector,
-    Physical,
-    Goal,
-    Magnet,
-    Seeker,
-    Player,
-    World,
-    Camp,
-)
+from .vector import Vector
+from .game import *
```

Comment:
when you do this, you also need to declare `__all__` in seekers.game

Reply:
I will add this.